projects/ddata-ui-input/src/lib/components/date/date-input.component.ts

```diff
@@ -99,16 +98,16 @@ export class DdataInputDateComponent implements OnInit {
   @Input() showIcon = true;
   @Input() autoApply = true;
   @Input() singleDatePicker = true;
-  @Input() locale: LocaleConfig = {
-    format: this.format,
-    displayFormat: this.format,
-    separator: this.separator,
-    cancelLabel: this.labelCancel,
-    applyLabel: this.labelApply,
-    daysOfWeek: this._moment.weekdaysMin(),
-    monthNames: this._moment.monthsShort(),
-    firstDay: this._moment.localeData().firstDayOfWeek(),
-  };
+  // @Input() locale: LocaleConfig = {
+  //   format: this.format,
+  //   displayFormat: this.format,
+  //   separator: this.separator,
+  //   cancelLabel: this.labelCancel,
+  //   applyLabel: this.labelApply,
+  //   daysOfWeek: this._moment.weekdaysMin(),
+  //   monthNames: this._moment.monthsShort(),
+  //   firstDay: this._moment.localeData().firstDayOfWeek(),
+  // };
```

Comment:
Ha nem kellenek már ezek ide, akkor töröld ki, de legyen megoldva, hogy a komponenst kívülről be lehessen állítani valamilyen nyelvre.